releasenotes/notes/48780.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+- 47423
+releaseNotes:
```

Comment:
nit: I'm a little worried that people may be confused because the wording and issue number on this are identical to the other https://github.com/istio/istio/pull/47538 which has already been included in https://istio.io/latest/news/releases/1.20.x/announcing-1.20.1/

maybe I'm over thinking it, or that people pay too close attention to our release notes.

Reply:
@GregHanson Yes, this may be a problem, but I'm not sure which option is the best since the old patch didn't fix the problem well. Alternatively, I could take a close eye on the new patch release and remove the note in 1.20.1 on the doc site when 1.20.3 is released, although maybe we didn't do this before,. I'm the RM for both 1.19 and 1.20, and the note should be put in the right place instead of dups.

Reply:
I'm OK with it going in as-is then if you're willing to adjust prior release notes

Reply:
Thanks